docs/admin/node-allocatable.md

```diff
@@ -9,115 +9,182 @@ title: Reserving Compute Resources for System Daemons
 * TOC
 {:toc}
 
-Kubernetes nodes can be scheduled to `Capacity`. Pods can consume all the available capacity on a node by default. This is an issue because nodes typically run quite a few system daemons that power the OS and Kubernetes itself. Unless resources are set aside for these system daemons, pods and system daemons compete for resources and lead to resource starvation issues on the node. 
-
-The `kubelet` exposes a feature named `Node Allocatable` that helps to reserve compute resources for system daemons. Kubernetes recommends cluster administrators to configure `Node Allocatable` based on their workload density on each node.
+Kubernetes nodes can be scheduled to `Capacity`. Pods can consume all the
+available capacity on a node by default. This is an issue because nodes
+typically run quite a few system daemons that power the OS and Kubernetes
+itself. Unless resources are set aside for these system daemons, pods and system
+daemons compete for resources and lead to resource starvation issues on the
+node.
+
+The `kubelet` exposes a feature named `Node Allocatable` that helps to reserve
+compute resources for system daemons. Kubernetes recommends cluster
+administrators to configure `Node Allocatable` based on their workload density
+on each node.
 
 ## Node Allocatable
 
 ```text
       Node Capacity
 ---------------------------
-|     kube-reserved		  |
+|     kube-reserved       |
 |-------------------------|
-|	  system-reserved	  |
+|     system-reserved     |
 |-------------------------|
-|    eviction-threshold	  |
+|    eviction-threshold   |
 |-------------------------|
-|						  |
-|  		allocatable		  |
-|  	    (available        |
-|  	   	 for pods)     	  |
-|  						  |
-|						  |
+|                         |
+|      allocatable        |
+|   (available for pods)  |
+|                         |
+|                         |
 ---------------------------
 ```
 
-`Allocatable` on a Kubernetes node is defined as the amount of compute resources that are available for pods.
-The scheduler does not over subscribe `Allocatable`.
-`CPU` and `memory` are supported as of now.
-Support for `storage` is expected to be added in the future.
+`Allocatable` on a Kubernetes node is defined as the amount of compute resources
+that are available for pods. The scheduler does not over subscribe
+`Allocatable`. `CPU` and `memory` are supported as of now. Support for `storage`
```

Comment:
the actual release version where storage will be added is not yet known.  depends on a number of factors.  we just know we will do more local storage management in the future.